modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -117,13 +122,12 @@
                 fieldMap[depend].fields.push(name)
             })
         })
-
+        
         /*
          * When a master is updated, refresh its slaves
          */
         $.each(fieldMap, function(fieldName, toRefresh){
-            fieldElements.filter('[data-field-name="'+fieldName+'"]')
-                .on('change.oc.formwidget', $.proxy(self.onRefreshDependants, self, fieldName, toRefresh))
+            $(document).on('change.oc.formwidget', '[data-field-name="' + fieldName + '"]', $.proxy(self.onRefreshDependants, self, fieldName, toRefresh));
```

Comment:
Can you explain this change more please?

Reply:
The repeater items can be added after the dom loaded. To add event listeners to items created after dom loaded you have to listen to document changes and filter it by field